app.py

```diff
@@ -0,0 +1,8 @@
+import connexion
```

Comment:
Should we move this to `mock_server/app.py`?

Reply:
Done.